compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -103,7 +103,9 @@ trait CommonScalaSettings:
   val pageWidth: Setting[Int] = IntSetting("-pagewidth", "Set page width", ScalaSettings.defaultPageWidth, aliases = List("--page-width"))
   val silentWarnings: Setting[Boolean] = BooleanSetting("-nowarn", "Silence all warnings.", aliases = List("--no-warnings"))
 
-  val release: Setting[String] = ChoiceSetting("-release", "release", "Compile code with classes specific to the given version of the Java platform available on the classpath and emit bytecode for this version.", ScalaSettings.supportedReleaseVersions, "", aliases = List("--release"))
+  val javaOutputVersion: Setting[String] = ChoiceSetting("-java-output-version", "version", "Compile code with classes specific to the given version of the Java platform available on the classpath and emit bytecode for this version. Corresponds to -release flag in javac.", ScalaSettings.supportedReleaseVersions, "", aliases = List("-release", "--release"))
```

Comment:
Could you make a PR to Scala 2 to also rename -release there?

Reply:
I'm not sure we should rename it for scala 2 as the semantics of -release and -target are slightly different there https://github.com/lampepfl/dotty/issues/8634#issuecomment-741909960

Reply:
Yeah I think that would also require changing the scala 2 flag to also set the target, IMO the current behavior is an oversight.

Reply:
https://github.com/scala/bug/issues/12543

without over-promising, maybe I'll take the opportunity to understand the issues and ergonomics.